mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -49,7 +54,20 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
             raise ValueError('target image cartesian image shape must be fill')
         raise NotImplementedError
     else:
-        data_thresholded = np.copy(data_values)
+        if data_values.ndim > 2:
+            warnings.warn('Data Values seems to have dimension '
+                          + str(data_values.ndim) +
+                          ' (>2). Using isFFT for now.')
+            isFFT = True
+        if isFFT:
+            img_shape = np.asarray(data_values[0].shape)
+            mask = convert_locations_to_mask(samples_locations, img_shape)
+            indices = np.where(np.reshape(mask, mask.size))[0]
+            data_thresholded = np.asarray([
+                np.reshape(data_values[ch], mask.size)[indices]
+                for ch in range(data_values.shape[0])])
```

Comment:
ch is a bit unclear as a variable name, does it stand for channel?

Reply:
Yes and Done